blog/_posts/2021-08-20-symbiosis-symbulation.md

```diff
@@ -0,0 +1,69 @@
+# Symbiosis in Symbulation
+#### August 5, 2021 Tiffany-Ellen Vo
+- - - 
+
+[comment]: <> (How do I make it so the text is aligned with the picture?)
+
+><img src="https://scontent-sjc3-1.xx.fbcdn.net/v/t1.6435-9/106543962_3538908449476892_6632988677477612051_n.jpg?_nc_cat=110&ccb=1-4&_nc_sid=09cbfe&_nc_ohc=MVm9XlfPXKYAX8k78kl&_nc_ht=scontent-sjc3-1.xx&oh=aff9f72aa230c94ddce73a0e5ab28c3e&oe=6131225E" alt="drawing" width="200"/> Hello! My name is Tiffany-Ellen Vo, I'm currently a senior at UC Santa Cruz, studying computer engineering. I’m especially interested in logic design, robotics, and computer architecture. In my free time, I love rock climbing, baking, and crocheting. To see some of my other projects, check out my GitHub!
+
+- - - 
+
+## Introduction to Symbiosis 
+  Through the guidance of my mentor Anya, this summer I have worked on simulating symbiosis; symbiosis is any type of close and long-term biological interaction between two different biological organisims which would be mutualistic, commensalistic, or parasitic. We can look at the movie _Finding Nemo_ (2003) for an example of symbiosis; the symbiotic relationship between an anemone and a clownfish is a classion one where both organisms benefit each other. The anemone provides the clownfish with shelter and protection, while the clownfish provides the anemone nutries in the form of waste and scares off potential predator fish. Symbiotic relationships are important because they are a major driving force of evolution. This networking and cooperation among species allows them to survive better than they would as individuals
+ 
+  While symbiosis has been simulated using Symbulation, the purpose of this project is to simulate symbiosis using the Empirical library in order to create and understand various types of symbiosis relationships while specificallly using the Avida complex genome to make the host and its symbionts.
```

Comment:
Symbulation was already using the Empirical library, so I would change this phrasing to emphasize the more complex genomes of Avida where the genomes are short computer programs.

Reply:
Should "symbiosis" be "symbiotic" here as well, in the phrase "understand various types of symbiosis relationships"?